.github/workflows/ci.yaml

```
@@ -15,47 +15,45 @@ permissions:
     contents: write # needed for mutex
 
 jobs:
+  get-values:
+    uses: ./.github/workflows/get-values.yaml
+    permissions:
+      contents: write # needed updating dependabot branches
+
   lint:
+    needs: [ get-values ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)